### ISSUE TITLE
Resource config forms' createElements(): don't return anything

### ISSUE THREAD

**Issue by Al2Klimov** [Icinga/icingaweb2#3048]
## Current Behavior

`createElements()` in resource config forms has to `return $this;` for the sake of *someone*'s lazyness.

## Expected Behavior

`createElements()` has to just to conform to our form interface definition.

## Reasons

* IDE autocompletion has not to suffer (`$form->createElements()->getElements()`)
* developers of future resource types (e.g. #2790, #3035) should have less to worry about

**Comment by Al2Klimov** [comment]
blocked by #3042

**Comment by Al2Klimov** [comment]
closing in favor of #3035
